stockpile/roles/dmidecode/files/dmidecode_dict.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+import re as _re
+from subprocess import check_output
+import json
+import sys
+
+# Simple test to ensure the dictionary is correctly built
+def verify_bios_vendor(dmidecode_input):
+    bios_vendor_value = dmidecode_input['BIOS_Information'][0]['Vendor']
+    args = ["dmidecode","--quiet","-s","bios-vendor"]
+    args = " ".join(args)
+    expected_bios_vendor = _execute_cmd(args)
+    if "".join(expected_bios_vendor.split()) == "".join(bios_vendor_value.split()):
```

Comment:
Can you add a comment describing when this false and what it means?  This is on the subtle side.